addons/test_website_modules/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# -*- encoding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Website Modules Test',
+    'version': '1.0',
+    'category': 'Hidden',
+    'sequence': 9876,
```

Comment:
tu es sur ;)?

Reply:
same as test_website, but I don't know how was the test_website one chosen :grimacing: